ngmix/priors.py

```diff
@@ -445,16 +625,12 @@ def get_fdiff_scalar(self, g1, g2):
         fdiffish = sqrt(chi2)
         return fdiffish
 
-    def get_fdiff_array(self, g1, g2):
-        """
-        For use with LM fitter, which requires
-        (model-data)/width
-        so we need to fake it
-
-        In this case the fake fdiff works OK because the prior
-        is on |g|, so the sign doesn't matter
-        """
-
+    def _get_fdiff_array(self, g1, g2):
+        # For use with LM fitter, which requires
+        #     (model-data)/width
+        # so we need to fake it
+        # In this case the fake fdiff works OK because the prior
+        # is on |g|, so the sign doesn't matter
         lnp = self.get_lnprob_scalar2d(g1, g2)
```

Comment:
I think I'm tired but I suspect this code never worked with arrays.

Reply:
I think you're right